src/MessageHandler/ActivityPub/Inbox/ActivityHandler.php

```diff
@@ -113,6 +117,14 @@ private function handle(?array $payload)
                     }
                 }
             }
+        } elseif ('Create' === $payload['type']) {
+            $actor = $this->manager->findActorOrCreate($payload['actor']);
+            if ($actor instanceof Magazine && $actor->lastOriginUpdate < (new \DateTime())->modify('-3 hours')) {
```

Comment:
This will not work, because the actor of a `Create` activity is never a magazine, but always a user. A magazine only announces what a user creates. If you get a plain `Create` activity you would have to check from which instance the user is from and then get the magazine the activity is posted to and then you can adjust the origin update. 
However I think I removed that part (I initially had something that did the same) because you cannot be sure that you get create activities because you are subscribed to a magazine, because you will get create activities from users you are subscribed to, regardless of whether you're subscribed to the magazine they post it to. `Announce` activities on the other hand are only sent to people subscribed to magazines